setup.py

```diff
@@ -26,19 +31,37 @@
 
 # NOTE: makefile cythonizes all Cython modules
 
+if USE_SYSTEM_DEPS:
```

Comment:
Alternatively, I could also add `and not NO_EXTENSIONS` here, to avoid calling pkg-config when we're not building extensions (though technically speaking, the other branch is also unnecessary then). That said, I'm not sure if we really need to special-case the case of setting `USE_SYSTEM_DEPS` when no extensions are built.

Reply:
I think this is fine. I don't think you would care about USE_SYSTEM_DEPS and `NO_EXTENSIONS` at the same time to warrant the additional complexity.